app/models/post.rb

```diff
@@ -0,0 +1,4 @@
+class Post < ApplicationRecord
+  belongs_to :user
+  validates :tittle, :body, presence: true
```

Comment:
# Mend the bug brought about by the validation.
- [ ] A user is unable to create a post successfully as the validation confuses rails. Let's fix this.
![Screenshot 2020-10-01 213325](https://user-images.githubusercontent.com/22345961/94849388-02634900-042e-11eb-8a24-a2169e2415ba.png)
```suggestion
  validates :title, :body, presence: true
```